build/protoc-go/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM golang:1.14
+FROM golang:1.16
 
 RUN apt-get update && \
     apt-get install -y unzip python3 python3-pip
 
-RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protoc-3.13.0-linux-x86_64.zip && \
-    unzip -o protoc-3.13.0-linux-x86_64.zip -d /usr/local bin/protoc && \
-    unzip -o protoc-3.13.0-linux-x86_64.zip -d /usr/local include/* && \
-    rm -rf protoc-3.13.0-linux-x86_64.zip
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.15.8/protoc-v3.15.8-linux-x86_64.zip && \
```

Comment:
it appears there's no `v` in `protoc-3.15.8-linux-x86_64.zip`
Should be
https://github.com/protocolbuffers/protobuf/releases/download/v3.15.8/protoc-3.15.8-linux-x86_64.zip

Reply:
🤦  .... certainly does. Thank you sir for the catch !